devel/libiodbc/Portfile

```diff
@@ -62,11 +64,61 @@ variant x11 {
     configure.args-delete   --disable-gui
 }
 
-default_variants +x11 +libodbc
+build.args
```

Comment:
FWIW, I don't believe `build.args` got set by this point, and I think it defaults to empty.

Reply:
I wasn't able to use `build.args` in the variant without setting this for some reason.